reinvent_models/lib_invent/models/dataset.py

```diff
@@ -74,5 +83,7 @@ def pad_batch(encoded_seqs):
     :param encoded_seqs: A list of encoded sequences.
     :return: A tensor with the sequences correctly padded.
     """
-    seq_lengths = torch.tensor([len(seq) for seq in encoded_seqs], dtype=torch.int64)  # pylint: disable=not-callable
+    seq_lengths = torch.tensor(
+        [len(seq) for seq in encoded_seqs], dtype=torch.int64
+    )  # pylint: disable=not-callable
     return (tnnur.pad_sequence(encoded_seqs, batch_first=True).cuda(), seq_lengths)

```

Comment:
aren't we missing the check here?

Reply:
I think the file was not saved that's why it was not pushed. I just pushed another change, it should be in those changes now.